README.md

```diff
@@ -51,11 +54,20 @@ Contributions are very welcome - please follow the [guidelines](CONTRIBUTING.md)
 - [ThermoML Archive](https://www.nist.gov/mml/acmd/trc/thermoml/thermoml-archive): experimental thermophysical and thermochemical property data (in ThermoML XML format)
 
 ## Target identification data
-- [Open Targets](https://platform.opentargets.org/): is a large-scale resource that uses human genetics and genomics data for systematic drug target identification and prioritization.
 
+- [Open Targets](https://platform.opentargets.org/): is a large-scale resource that uses human genetics and genomics data for systematic drug target identification and prioritization.
 - [Probes & Drugs Portal](https://www.probes-drugs.org/data_sources): is an interactive, open data resource for chemical biology. Overview of libraries of bioactive compounds (e.g., ChEMBL, Guide to PHARMACOLOGY), including commercial screening libraries.
 
+## Pharmacology & ADME & Metabolism
+
 - [Guide to PHARMACOLOGY](https://www.guidetopharmacology.org/): is an expert-curated resource of ligand-activity-target relationships. It includes activity data even for data with unknown bioactivity value (under CC BY-SA 4.0).
+- [Drug Indications Database(DID)](https://doi.org/10.1186/s13326-016-0110-0): is a data of structured drug-indication relations. It is intended to facilitate building practical, comprehensive, integrated drug ontologies.
+- [The Metabolism and Transport Database ](https://www-metrabase.ch.cam.ac.uk/metrabaseui/pageview/download/):  is a cheminformatics and bioinformatics resource that contains curated data related to human small molecule metabolism and transport.
+- [The Human Metabolome Database (HMDB)](https://hmdb.ca/):  is a freely available electronic database containing detailed information about small molecule metabolites found in the human body.
+- [KEGG PATHWAY Database(KEGG)](https://www.genome.jp/kegg/kegg2.html): a database resource for understanding high-level functions and utilities of the biological system, such as the cell, the organism and the ecosystem, from molecular-level information, especially large-scale molecular datasets generated by genome sequencing and other high-throughput experimental technologies.
```

Comment:
Do you know if there is a way to download this data in a machine-readable form? I clicked on a few links and saw a lot of images

Reply:
Yeah metabolism data have even problem in readability. We will face a challenge in dealing with this data, best things to do is to make it available until we find away to deal with or use it in another way. It is Not worth mentioning.